src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -218,6 +280,13 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
             this.emit(MatrixRTCSessionEvent.MembershipsChanged, oldMemberships, this.memberships);
         }
 
+        const ts = (m: CallMembership): string => `${m.member.userId}:${m.deviceId}`;
+        const callMembersChanged = new Set(oldMemberships.map(ts)) !== new Set(this.memberships.map(ts));
```

Comment:
I'm not sure I understand how this is different to what's just been done above.

Reply:
We don't want to send a new event if the content of the membership changes - we only care about joins/leaves